docs/tokens/emptystate.mdx

```diff
@@ -0,0 +1,18 @@
+---
+name: Empty State
+menu: Tokens
+
+---
+import { Playground, PropsTable } from 'docz'
+import emptyState from '../../resources/images/illustrations/error-404.svg'
+
+# Empty State
+
+<Playground>
+    <div class="card_zero_result">
```

Comment:
class name should follow the BEM conventions.

examples

```
<div class="blank-slate">
   <div class="blank-slate-content">
      <div class="blank-slate-content__illustration">
      </div>
      <div class="blank-slate-content__message"></div>
      <div class="blank-slate-content__cta"></div>
   </div>
</div>
</div>
```

Reply:
https://success.outsystems.com/Documentation/11/Developing_an_Application/Design_UI/Patterns/Using_Traditional_Web_Patterns/Content/BlankSlate

Reply:
https://styleguide.github.com/primer/components/blankslate/